k8s/namespaces/probate/probate-cron-smee-and-ford-extract/aat-00.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: probate-cron-smee-and-ford-extract
+spec:
+  releaseName: probate-cron-smee-and-ford-extract
+  values:
+    global:
+      jobKind: CronJob
+      enableKeyVaults: true
+      tenantId: "531ff96d-0ae9-462a-8d2d-bec7c0b42082"
+      environment: aat
+    job:
+      schedule: 0/10 * * * *
```

Comment:
are you sure you want to try all these for every 10mins?

Reply:
I'm going to change it to nightly once we have tested this scheduler

Reply:
ok - be sure to keep an eye on the pod + revert asap